fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Utils.java

```diff
@@ -370,8 +372,16 @@ public static boolean hasUnknownColumnsStats(OptExpression root) {
                     scanOperator.getColRefToColumnMetaMap().values().stream().map(Column::getName).collect(
                             Collectors.toList());
             if (operator instanceof LogicalOlapScanOperator) {
+                Table table = scanOperator.getTable();
```

Comment:
I think need to distinguish key columns and value columns for aggregate table.

Reply:
Seems hard to making precise judgments

Reply:
ColumnRefFactory could get Column， Column has `isKey` method

Reply:
can be check aggreagte key through ScanNode.columnMetaToColRefMap @liuyehcf